format.sh

```diff
@@ -1,12 +0,0 @@
-#!/bin/bash
-
-if [[ -z $1 ]]; then
-  echo "Error: expected args"
-  exit 1
-fi
-node_modules/.bin/prettier \
```

Comment:
Moved the common prettier config to package.json so no need for this file anymore